internal/tui/nomad/jobadmin.go

```diff
@@ -0,0 +1,64 @@
+package nomad
+
+import (
+	"fmt"
+
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/hashicorp/nomad/api"
+
+	"github.com/robinovitch61/wander/internal/tui/style"
+)
+
+var (
+	JobAdminActions = map[AdminAction]string{
+		StopJobAction:   "Stop",
+		StopAndPurgeJobAction:   "Stop and purge",
+	}
+)
+
+type JobAdminActionCompleteMsg struct {
+	Err     error
+	JobID string
+}
+
+func GetJobAdminText(adminAction AdminAction, jobID string) string {
+	switch adminAction {
+	case StopJobAction, StopAndPurgeJobAction:
+		return fmt.Sprintf(
+			"%s job %s",
+			JobAdminActions[adminAction],
+			style.Bold.Render(jobID))
+	default:
+		return "----"
```

Comment:
```suggestion
		return ""
```

For consistency with allocadmin